test/mail_helper.js

```diff
@@ -52,8 +52,12 @@ module.exports = (printLogs) => {
               const smsUrlMatch = /(https?:\/\/.*$)/.exec(mail.text)
               smsLink = smsUrlMatch && smsUrlMatch[1]
             }
+            
+            // Workaround because the email service wraps this header in `< >`.
+            // See: https://github.com/mozilla/fxa-content-server/pull/6470#issuecomment-415224438
+            var name = emailName(mail.headers.to.replace(/\<(.*?)\>/g, '$1'))
+            if (mail.headers['x-email-service'] === 'fxa-email-service' && mail.headers.to.charAt(0) === '<') {
```

Comment:
do you still need this?

Reply:
WOW, that was careless of me to live it there, sorry. It's fixed.